src/main/java/com/salesforce/cdp/queryservice/auth/jwt/JwtCoreTokenProvider.java

```diff
@@ -0,0 +1,78 @@
+package com.salesforce.cdp.queryservice.auth.jwt;
+
+import com.salesforce.cdp.queryservice.auth.*;
+import com.salesforce.cdp.queryservice.model.CoreTokenRenewResponse;
+import com.salesforce.cdp.queryservice.util.Constants;
+import com.salesforce.cdp.queryservice.util.TokenException;
+import lombok.extern.slf4j.Slf4j;
+
+import java.sql.SQLException;
+import java.util.Optional;
+import java.util.Properties;
+
+import static com.salesforce.cdp.queryservice.util.Messages.TOKEN_FETCH_FAILURE;
+
+@Slf4j
+public class JwtCoreTokenProvider implements CoreTokenProvider {
+
+    private final Properties properties;
+    private final JwtLoginClient client;
+
+    private Optional<OffcoreToken> offcoreToken;
```

Comment:
What is use of `offcoreToken` field in this class?

Reply:
This is not required. This is something which got missed